src/packages/utils/src/utils/constants.ts

```diff
@@ -15,4 +15,4 @@ export const RPCQUANTITY_ONE = Quantity.from(1n);
 export const RPCQUANTITY_GWEI = Quantity.from(1000000000);
 export const WEI = 1000000000000000000n as const;
 
-export const KNOWN_CHAINIDS = new Set([1, 3, 4, 5, 42]);
+export const KNOWN_CHAINIDS = new Set([1, 3, 4, 5, 42, 11155111]);
```

Comment:
`11155111` is the `sepolia` chain's `ChainID`.

Might be worth noting that we are NOT adding this to our list of Infura known networks because Infura doesn't know about it right now.